app/helpers/application_helper.rb

```diff
@@ -2,4 +2,8 @@ module ApplicationHelper
   def team_logo(image_path)
     cl_image_tag(image_path, { size: '32x32', crop: :fill, radius: 5, border: '1px_solid_black' })
   end
+
+  def user_gravatar(user)
```

Comment:
Also, call it `user_avatar` instead of `gravatar`, do not couple the helper name with an external service.